Automattic/wp-calypso
# Marketplace: Combine collections and categories

#### Proposed Changes

* Moves `useCollections` into `useCategories`
* Split category name into category menu (and nav) text and title text shown on the page. This was required to mix ecommerce category and ecommerce collection appropriately
* Added menu titles for paid/popular/featured so it shows up in the breadcrumbs in a shorter / more appropriate form (Had to make some assumptions here but I think it looks ok)
* Enables browse all links and points them at category pages
* Split business tags category out from ecommerce category + dropped business directory tag, didn't look too helpful.

#### Testing Instructions

* View /plugins
* Should see the same collections and sections as before with the same titles
* Browse all links should now be available on collections and they should take you to an appropriate browse page
* Category pages should now render the title and description on the page but the menu text should be shown in the breadcrumb and the menu selector.
* Ecommerce category page should have a new title and description 
* Ecommerce category page should have the old title "Ecommerce & Business" seen in breadcrumbs and the menu selector

Category page: `/plugins/browse/ecommerce`
![Screenshot(40)](https://user-images.githubusercontent.com/811776/195770388-24e8b536-6636-49b8-8955-506952a894a3.png)

Discover page: `/plugins`
![Screenshot(39)](https://user-images.githubusercontent.com/811776/195770392-7d003970-10dd-46db-9cb2-3637e00ffc20.png)


Fixes #68945
Fixes #68943
